src/lib/src/component/navigation/navigation.component.css

```diff
@@ -54,6 +54,16 @@
   cursor: pointer;
 }
 
-.icon:hover {
+.icon:hover:enabled {
   color: #ffffff;
 }
+
+:disabled {
```

Comment:
Is this for disabled elements? Is that wanted?

Reply:
Shouldn't it be `.icon:disabled` ?